receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_scraper_windows_test.go

```diff
@@ -55,8 +57,10 @@ func TestScrape_Errors(t *testing.T) {
 			getPageFileStats: func() ([]*pageFileStats, error) {
 				return []*pageFileStats{testPageFileData}, nil
 			},
-			expectedUsedValue: int64(testPageFileData.usedBytes),
-			expectedFreeValue: int64(testPageFileData.freeBytes),
+			expectedUsedValue:            int64(testPageFileData.usedBytes),
+			expectedFreeValue:            int64(testPageFileData.freeBytes),
+			expectedUtilizationFreeValue: 80.0,
+			expectedUtilizationUsedValue: 20.0,
```

Comment:
Why we have percents here? How does the test pass?

Reply:
that's true! in fact I've just seen that the tests are failing when running from my machine. I think they don't get executed in the pipeline.
I've pushed a fix for the test